src/modules/views/AgentDetails.jsx

```diff
@@ -14,36 +16,39 @@ const useStyles = makeStyles(() => ({
     justifyContent: "spaceAround",
   },
   card: {
-    height: "450px",
+    height: "auto",
     boxShadow: "0 10px 25px 5px rgba(0, 0, 0, 0.2)",
     overflow: "hidden",
     backgroundColor: "#3e4453",
   },
   cardHeader: {
     position: "relative",
     paddingTop: "50px",
-    paddingBottom: "50px",
+    paddingBottom: "20px",
```

Comment:
I do think we should leave the height how it was in case we want to use the top div as a color accent. Otherwise, the background color can be changed in the "card" class. That's the only reason cardHeader was there originally.

Reply:
Yeah I'd be down for that, but would be better to use `em` or `%` for responsiveness over `px`

Reply:
Definitely agree